rosplan_planning_system/include/rosplan_action_interface/RPSimulatedActionInterface.h

```diff
@@ -11,7 +11,7 @@
  * This file defines the RPMoveBase class.
  * RPMoveBase is used to connect ROSPlan to the MoveBase library.
  * PDDL "goto_waypoint" actions become "move_base_msgs::MoveBase" actions.
- * Waypoint goals are fetched by name from the SceneDB (implemented by mongoDB).
```

Comment:
Is this comment correct? Looks like it was copied from another class.

Reply:
True.. I will remove it entirely. Thanks